browser/importer/brave_external_process_importer_client.cc

```diff
@@ -14,7 +14,7 @@
 
 namespace {
 bool ShouldUseBraveImporter(importer::ImporterType type) {
-  if (type == importer::TYPE_CHROME)
+  if (type == importer::TYPE_CHROME || type == importer::TYPE_EDGE)
```

Comment:
Per other comments, we can update to be `importer::TYPE_CHROMIUM_EDGE`

Reply:
done